debug_gtfs_graph.rb

```diff
@@ -0,0 +1,39 @@
+ActiveRecord::Base.logger = Logger.new(STDOUT)
```

Comment:
Is there a better place to put this than the root? Or is it a temporary test script that's better not to commit to the repo?